third_party/blink/renderer/includes.gni

```diff
@@ -1,29 +1,27 @@
 # common includes which can help minimize patches for
 # src/third_party/blink/renderer/core/BUILD.gn
 brave_blink_renderer_core_visibility = [
-  "//brave/third_party/blink/renderer/*"
+  "//brave/third_party/blink/renderer/*",
+  "//brave/components/brave_search/renderer:renderer",
```

Comment:
why is this being added to blink visibility? Is this maybe leftover from previous patching?

Reply:
redid the mojo and removed it from here